test/functional/CacheManagement/src/tests/sharedclasses/options/TestSharedCacheJavaAPI.java

```diff
@@ -66,8 +88,41 @@ public static void main(String args[]) {
 		    if (oldCacheCount == -1) {
 		    	fail("iterateSharedCacheFunction failed");
 		    }
-		    persistentCount = nonpersistentCount = snapshotCount = 0;
+	    	
+	    	if (dir == null && false == isWindows() && false == isMVS() && isOpenJ9()) {
+	    		dirGroupAccess = getCacheDir("Foo_groupaccess", false);
+	    		dirRemoveJavaSharedResources = removeJavaSharedResourcesDir(dirGroupAccess);
+	    		cacheGroupAccessList = SharedClassUtilities.getSharedCacheInfo(dirGroupAccess, SharedClassUtilities.NO_FLAGS, false);
+		    	if (cacheGroupAccessList == null) {
+		    		oldCacheGroupAccessCount = 0;
+		    	} else {
+		    		oldCacheGroupAccessCount = cacheList.size();
```

Comment:
Should this be cacheList.size() or cacheGroupAccessList.size() ?

Reply:
Correct! Updated.